### PR TITLE
LPS-198743 Add redirection to display page template on success object actions

### PR DESCRIPTION
**Pull request by claraizquierdo**
**IMPORTANT!!!** THIS PR DEPENDS ON https://github.com/liferay-echo/liferay-portal/pull/14147

Motivation
=======
[Link to story or ticket](https://liferay.atlassian.net/browse/LPS-198743)

Steps to Verify:
----------------
1. Enable feature flags feature.flag.LPS-195263=true and feature.flag.LPS-195205
1. Create an Object with one action
1. Create an instance of that object
2. Create a display page template for the Object just created
1. Go to the page editor a add a button fragment
1. In the configuration panel of the button select Type Action
3. Select the action in the elements tree and in its configuration panel, in the action tab, select on success interaction "Go to Entry Display Page"
4. Select a display page

Screenshots (optional):
-----------------------
Captura de pantalla 2023-10-30 a las 17 51 10
here

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/EditableActionPanel.js [hunk 1]
```diff
@@ -62,13 +62,16 @@ const ERROR_INTERACTION_OPTIONS = [
...
 
-const SUCCESS_INTERACTION_OPTIONS = [
-	...ERROR_INTERACTION_OPTIONS,
-	{
-		label: Liferay.Language.get('go-to-entry-display-page'),
-		value: INTERACTION_DISPLAY_PAGE,
-	},
-];
+let SUCCESS_INTERACTION_OPTIONS = ERROR_INTERACTION_OPTIONS;
+if (Liferay.FeatureFlags['LPS-195263']) {
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/EditableActionPanel.js:66]: Silly thing, but I'd put a blank line before the if statement

File: modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/plugins/browser/components/page_structure/components/item_configuration_panels/EditableActionPanel.test.js [hunk 2]
```diff
@@ -209,4 +209,20 @@ describe('EditableActionPanel', () => {
...
+			state: getStateWithConfig({
+				mappedAction: {...MAPPED_ACTION, fieldId: 'actionFieldId'},
+				onSuccess: {interaction: 'displayPage'},
+			}),
+		});
+
+		expect(screen.getByText('display-page')).toBeInTheDocument();
+		expect(
+			screen.queryByText('reload-page-after-success')
+		).not.toBeInTheDocument();
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/test/page_editor/app/plugins/browser/components/page_structure/components/item_configuration_panels/EditableActionPanel.test.js:226]: We should disable the FF after the test. Maybe wrapping the whole test in a `describe` with `beforeAll` and `afterAll`

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/EditableActionPanel.js [hunk 3]
```diff
@@ -239,9 +239,22 @@ function InteractionSelector({
...
 	};
 
-	const mappingIds = collectionConfig
-		? collectionConfig.collection
-		: config.mappedAction;
+	let mappingIds = null;
+
+	// If inside a DisplayPageTemplate
+
+	if (itemConfig.mappedAction.mappedField) {
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/plugins/browser/components/page_structure/components/item_configuration_panels/EditableActionPanel.js:246]: Not a fan of this kind of comments. It looks like `itemConfig.mappedAction.mappedField` is not related at all with DisplayPages.

**Review by p2kmgcl — COMMENTED** [review]
Just added a few comments to discuss.

**Comment by claraizquierdo** [comment]
@ruben-pulido I've included the field rename here

**Comment by p2kmgcl** [comment]
ci:forward the path to glory

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by Tim-Cao** [comment]
ci:forward:force

**Pull request closed without merging**